### PR TITLE
Update CI status badge in readme

### PR DESCRIPTION
**Pull request by pierlon**
## Summary

Replaces Travis CI status badge with the status badge for the "Build, test & measure" GitHub Action workflow.

## Checklist

- [ ] My pull request is addressing an open issue (please create one otherwise).
- [ ] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [ ] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -14,5 +14,5 @@ indent_style = tab
 indent_style = space
 indent_size = 2
 
-[{*.md}]
+[*.md]
```
Comment (pierlon) [inline .editorconfig:17]: This rule didn't seem to be matching as PHPStorm kept removing the trailing whitespace. Also, the linter was giving the error:

image

Once the braces were removed PHPStorm stopped removing the trailing whitespace.

File: readme.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ The Official AMP Plugin, supported by the AMP team. Formerly Accelerated Mobile
 **License:** [GPLv2 or later](http://www.gnu.org/licenses/gpl-2.0.html)  
 **Requires PHP:** 5.6  
 
-[![Build Status](https://travis-ci.org/ampproject/amp-wp.svg?branch=develop)](https://travis-ci.org/ampproject/amp-wp) [![Coverage Status](https://img.shields.io/codecov/c/github/ampproject/amp-wp/develop.svg)](https://codecov.io/gh/ampproject/amp-wp) [![Built with Grunt](https://gruntjs.com/cdn/builtwith.svg)](http://gruntjs.com) 
+[![Build Status](https://github.com/ampproject/amp-wp/workflows/Build,%20test%20&%20measure/badge.svg)](https://github.com/ampproject/amp-wp/actions?query=branch%3Adevelop+workflow%3A%22Build%2C+test+%26+measure%22) [![Coverage Status](https://img.shields.io/codecov/c/github/ampproject/amp-wp/develop.svg)](https://codecov.io/gh/ampproject/amp-wp) [![Built with Grunt](https://gruntjs.com/cdn/builtwith.svg)](http://gruntjs.com)
```
Comment (pierlon) [inline readme.md:15]: At the moment the badge label is "Build, test & measure". Customizing the label is not possible but it has been reported on GitHub's community forums as a [requested feature](https://github.community/t/let-us-configure-the-badge-labels/138418).

image

Comment (pierlon) [reply readme.md:15]: Also, the status is for the default branch (`develop`).

**Review by westonruter — APPROVED** [review]

**Comment by westonruter** [comment]
I updated the require status checks to pass before merging to remove the space.

Just E2E is failing, probably for no good reason. I wonder: should we allow E2E to fail (for now) and make it not required since it seems extremely fragile and we plan to improve the infrastructure used to run it?

**Comment by pierlon** [comment]
> should we allow E2E to fail (for now) and make it not required since it seems extremely fragile and we plan to improve the infrastructure used to run it?

We could, but I'd recommend keeping a close eye on it as Gutenberg does ship breaking changes from time to time.

**Comment by pierlon** [comment]
Marked E2E job as allowed to fail in https://github.com/ampproject/amp-wp/pull/5583/commits/05d55f210dfe2a180b7dc0485079ca457f234c7f.

**Comment by westonruter** [comment]
We currently have two jobs for PHPUnit with PHP 7.4, one with coverage and one without:

image

Is this not redundant?

**Comment by pierlon** [comment]
The one with coverage supplies Coveralls with the PHP coverage. Previously with Travis that job only ran on the `develop` branch.

**Comment by pierlon** [comment]
I suppose we could get rid of the non-coverage one.

**Comment by pierlon** [comment]
Ah wait, the non-coverage one also ran the `external-http` tests, while the one with coverage did not.

**Comment by westonruter** [comment]
Right, because if the version of PHP is the same and it's doing the same tests, but one has coverage and another has none, then we should be able to skip the non-coverage one.

**Pull request merged**